Makefile

```diff
@@ -26,8 +26,10 @@ vet:
 install-tools:
 	@echo ">> fetching goimports"
 	@go get -u golang.org/x/tools/cmd/goimports
+	@go install golang.org/x/tools/cmd/goimports
```

Comment:
I don't get that. `go get` always do `go install` at then end, unless `-d` option is set. So this change should be not needed.